public/app/plugins/datasource/influxdb/specs/response_parser.test.ts

```diff
@@ -306,13 +306,16 @@ describe('influxdb response parser', () => {
 
     const fetchMock = jest.spyOn(backendSrv, 'fetch');
 
+    const annotation = {
```

Comment:
What's the purpose of this hardcoded annotation?

Reply:
This was hard coded to explicitly show that we are leveraging the `targets` of the query to handle annotations now